bedrock/firefox/templates/firefox/dev-firstrun.html

```diff
@@ -92,10 +97,9 @@
           <div class="video" data-video-id="eH0R10Ga4Hs"></div>
         </div>
       </li>
-    {% endif %}
       <li class="feature">
         <a href="https://www.youtube.com/watch?v=eQqNfkqIJdw" rel="external" class="video-play">
-          <img src="{{ static('img/firefox/firstrun/dev/feature-inspector-large.jpg') }}" alt="{{ _('Screenshot') }}" class="screenshot">
+          <img src="{{ static('img/firefox/firstrun/dev/feature-inspector.png') }}" alt="{{ _('Screenshot') }}" class="screenshot">
```

Comment:
I think we need to keep this as `feature-inspector-large.jpg`, as otherwise the image sizes are mismatched:

<img width="896" alt="screen shot 2015-10-30 at 4 24 46 pm" src="https://cloud.githubusercontent.com/assets/317532/10858405/c4aceb6a-7f22-11e5-8d57-5eed4a5944b8.png">